main.js

```diff
@@ -0,0 +1,177 @@
+const player1 = document.querySelector("#player1");
```

Comment:
You could wrap all of your code inside a IIFE to take everything out from the global scope:

```js
(() => {
  const player1 = document.querySelector("#player1");

  // ...

  form.addEventListener("submit", event => {
    event.preventDefault();
    congrats.innerHTML = "";
    //Clear board
    DisplayController.clearBoard();

    //Set all value in array to empty
    DisplayBoard.gameBoard = ["", "", "", "", "", "", "", "", ""];
    GameFlow.changeGameStatus();
    GameFlow.start();
  });
})();
```